tests/properties/pflua_pipelines_match.lua

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env luajit
+-- -*- lua -*-
+module(..., package.seeall)
+package.path = package.path .. ";../?.lua;../../src/?.lua"
+-- Compare the results of the libpcap/bpf and pure-lua pflua pipelines.
+
+local pf = require("pf")
+local savefile = require("pf.savefile")
+local utils = require('pf.utils')
+
+local pflang = require('pfquickcheck.pflang')
+
+function property(packets)
+   --nil pkt_idx, pflang_expr, bpf_result, pflua_result to avoid
+   -- confusing debug information
+   pkt_idx, pflang_expr, bpf_result, pflua_result = nil
+   local pkt, P, pkt_len, libpcap_pred, pflua_pred
+   a = pflang.Pflang()
+   pflang_expr = table.concat(a, ' ')
```

Comment:
The generator is intentionally ignoring all of that for the moment, partly to mimic pflang expressions that may be generated by people not thinking about precedence rules but which should still behave the same way under pflua or libpcap. It doesn't have the same constraints with respecting precedence rules as most tools would, as long as it's not generating outright invalid pflang.